### PR TITLE
v.01

### PR DESCRIPTION
**Pull request by nazar-medushevskyi**
- [DEMO LINK](https://nazar-medushevskyi.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+@import "./utils/var";
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:1]: Add this file to commit. I don't see it

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    border: 1px solid #000;
+    font-style: "Arial", sans-serif;
+    font-size: 30px;
+    cursor: pointer;
+    background-color: #eee;
+    transition: background-color 0.5s,
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:28]: make variable for 0.5s

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+@import "./utils/var";
+
+* {
+  box-sizing: border-box;
+}
+
+.calendar {
```
Comment (DarkMistyRoom) [inline src/styles/main.scss:7]: image

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
image
image

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Comments that was left by previous reviewer is not fixed. Please fix it before re-request review. If you have any questions feel free to sak in Slack

**Pull request closed without merging**